waspc/test/Psl/Parser/ConfigBlockTest.hs

```diff
@@ -25,7 +26,7 @@ spec_parsePslConfigBlock = do
             Psl.ConfigBlock.ConfigBlock
               Psl.ConfigBlock.Datasource
               "db"
-              [ Psl.ConfigBlock.KeyValuePair "provider" $ Psl.Argument.StringExpr "postgresql",
+              [ Psl.ConfigBlock.KeyValuePair "provider" $ Psl.Argument.StringExpr (AS.Db.providerToString AS.Db.PostgreSQL),
```

Comment:
Similar argument here as far above -> if everything else here is hardcoded, like `db`, and `env`, and `url`, and simlar, then you can also hardcode these values, doesn't make much sense to have them parametrized and the rest not. It is ok, there are tests of Prisma parser, we can assume prisma syntax here.